.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -95,6 +99,17 @@ jobs:
         export WINDOWS_E2E_IMAGE=${{ parameters.windowsImage }}
         export WINDOWS_DISTRO=${{ parameters.windowsDistro }}
         export WINDOWS_NODE_IMAGE_VERSION=${{ parameters.windowsNodeImageVersion }}
+        export WINDOWS_GPU_DRIVER_SUFFIX=${{ parameters.windowsGpuDriverSuffix }}
+        if [[ "${WINDOWS_GPU_DRIVER_SUFFIX}" == "-grid" ]]; then
+            export WINDOWS_GPU_DRIVER_URL="https://acs-mirror.azureedge.net/aks/windows/gpudrivers/grid/537.13_grid_win10_win11_server2019_server2022_dch_64bit_international_azure_swl.exe"
```

Comment:
Can we update the driver versions from a configuration file instead of this pipeline?

Reply:
Can I create two params in e2e-windows-template.yaml, then set it and pass it from e2e-windows.yaml?

Reply:
Or do you mean that we can create a centralized template file to update the driver urls in only one place? Something like https://github.com/Azure/azhpc-extensions/blob/master/NvidiaGPU/resources.json used by the vm extension?

Reply:
I think that we should define the gpu drivers in agentbaker to avoid failures caused by external changes.

Reply:
I think the main concern here is how to align the driver versions for test with the driver versions in prod environment. We should avoid to update the urls manually every time the new driver versions are released.

Reply:
Just added a task to track this issue. This issue will be resolved later. 

https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/25951813